### ISSUE TITLE
"Element with role attribute has required states and properties" [4e8ab6] should accept closed `combobox` without `aria-controls`

### ISSUE THREAD

**Issue by Jym77** [auto-wcag/auto-wcag#2029]
As a follow-up to some of the changes in #2012…

In ARIA 1.2, `combobox` requires `aria-controls`, this has been removed in ARIA 1.3 and the authoring practice say it's only needed when the combobox is expanded (more details:https://github.com/act-rules/act-rules.github.io/pull/2012#discussion_r1081026609)

Therefore, [Element with role attribute has required states and properties](https://www.w3.org/WAI/standards-guidelines/act/rules/4e8ab6/proposed/) should also have an exception for "missing" `aria-controls` on `combobox` with `aria-expanded="false"`.
